data-prepper-core/src/main/java/org/opensearch/dataprepper/parser/PipelineTransformer.java

```diff
@@ -5,6 +5,9 @@
 
 package org.opensearch.dataprepper.parser;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
```

Comment:
I think that all of these changes are obsolete and that you can revert them.

Reply:
addressed in the latest pr